application/F3DStarter.cxx

```diff
@@ -165,6 +166,195 @@ class F3DStarter::F3DInternals
     }
   }
 
+  void addOutputImageMetadata(f3d::image& image)
```

Comment:
I wonder what other metadata we should add ? 
Once we adresse #596 , should we store everything ?

Reply:
Very good point I did not consider. In any case, out of scope of this PR.